packages/usa-validation/src/index.js

```diff
@@ -1,14 +1,57 @@
 const behavior = require("../../uswds-core/src/js/utils/behavior");
 const validate = require("../../uswds-core/src/js/utils/validate-input");
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+
+const VALIDATE_INPUT = "input[data-validation-element]";
+const CHECKLIST_ITEM = `.${PREFIX}-checklist__item`;
 
 function change() {
   validate(this);
 }
 
-const validator = behavior({
-  "keyup change": {
-    "input[data-validation-element]": change,
+const enhanceValidation = (target) => {
+  selectOrMatches(VALIDATE_INPUT, target).forEach((input) => {
+    const validationParent = input.parentNode;
+    const checklistItems =
+      validationParent.querySelectorAll(CHECKLIST_ITEM);
+
+    // Set up input attributes
+    input.setAttribute("aria-controls", "validate-code");
+
+    // Create container to hold aria readout
+    const statusSummaryContainer = document.createElement("span");
+    statusSummaryContainer.setAttribute("data-validation-status", "");
+    statusSummaryContainer.classList.add("usa-sr-only");
+    statusSummaryContainer.setAttribute("aria-live", "polite");
+    statusSummaryContainer.setAttribute("aria-atomic", true);
+    validationParent.append(statusSummaryContainer);
```

Comment:
This could be moved to its own function.

```js
const createStatusElement = (validationContainer) => {
  const statusSummaryContainer = document.createElement("span");
  
  statusSummaryContainer.setAttribute("data-validation-status", "");
  statusSummaryContainer.setAttribute("aria-live", "polite");
  statusSummaryContainer.setAttribute("aria-atomic", true);

  statusSummaryContainer.classList.add("usa-sr-only");

  validationContainer.append(statusSummaryContainer);
}
```
```

Reply:
Wouldn't this status element have to be tied to the input as well? If so, we'd have to give it a unique ID and add it to the input's `aria-describedby`.

I didn't notice any major issues with it being read in voiceover. On rare occasion it would either cutoff the status announcement or not give it at all.